tools/mksurfdata_esmf/src/mklanwatMod.F90

```diff
@@ -253,12 +258,11 @@ subroutine mklakdep(file_mesh_i, file_data_i, mesh_o, pioid_o, fsurdat, rc)
     if (chkerr(rc,__LINE__,u_FILE_u)) return
 
     ! Create a route handle between the input and output mesh
-    if (.not. ESMF_RouteHandleIsCreated(routehandle_mklak)) then
```

Comment:
Right. This may have been needed when the same routehandle was used for both lake areas and the lake depth parameter: then, for transient datasets, we wanted to make sure that the routehandle was only created once even though it was used many times. Now I have separated the routehandle used for lake areas from the one used for lake depth, because the mappings are now done differently. So you'll notice that I introduced a new local variable for routehandle (line 198) and, even in a transient run, this routine is just called once, so this local routehandle is created when the routine is called and just used the one time.

Reply:
Thanks for that explanation. That makes perfect sense to me.